metagraph/src/cli/config/config.cpp

```diff
@@ -774,7 +774,11 @@ void Config::print_usage(const std::string &prog_name, IdentityType identity) {
             fprintf(stderr, "\t   --graph [STR] \tgraph representation: succinct / bitmap / hash / hashstr / hashfast [succinct]\n");
             fprintf(stderr, "\t   --count-kmers \tcount k-mers and build weighted graph [off]\n");
             fprintf(stderr, "\t   --count-width \tnumber of bits used to represent k-mer abundance [8]\n");
+#if _PROTEIN_GRAPH
+            fprintf(stderr, "\t   --index-ranges [INT]\tindex all node ranges in BOSS for suffixes of given length [3]\n");
+#else
             fprintf(stderr, "\t   --index-ranges [INT]\tindex all node ranges in BOSS for suffixes of given length [10]\n");
+#endif
```

Comment:
Better generalize this to `log(2^20)/log(|Sigma|)`, so for DNA with |Sigma| = 4, this will be exactly 10, and 4 for Proteins.